# Homework 1 Problem 2

In [1]:
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt

In [2]:
from scipy.constants import e, m_e, k, pi, epsilon_0, c

temperature = 273
concentration = 8.47 * 10 ** 28

In [3]:
def plasma_frequency(concentration):
    return np.sqrt((concentration * e ** 2) / (m_e * epsilon_0))

In [4]:
omega_p = plasma_frequency(concentration)
omega_p

1.6418490871041346e+16

$\omega_p = 1.64185 \cdot 10^{16} s^{-1}$

In [5]:
def tau(density, resistivity):
    return m_e / (e ** 2 * density * resistivity)

tau_ = tau(density = 8.47 * 10 ** (28), resistivity = 1.56 * 10 ** (-8))

In [6]:
tau_

2.685716701399944e-14

In [7]:
tau_ * omega_p

440.9541514413826

$\tau \cdot \omega_p \approx 500 >> 1$

In Drude model, if $\omega_p \tau >> 1$, dielectric constant is given by:

\begin{equation}
\epsilon(\omega) = 1 - \frac{\omega_p^2}{\omega^2}
\end{equation}

For frequencies $\omega >> \omega_p$, we have $\epsilon(\omega) \approx 1$, therefore metal is transparent.

Equivalently, for $\lambda << \lambda_p = \frac{2 \pi c}{\omega_p}$, metal is transparent

In [8]:
2 * pi * c / omega_p

1.1472744858854266e-07

Therefore, limiting values are:

$\omega_p = 1.64185 \cdot 10^{16} s^{-1}$

$\lambda_p = 1.14727 \cdot 10^{-7} m$